### PR TITLE
Refactor API Calls

### PR DESCRIPTION
**Pull request by alexander-mathieu**
This PR creates a googleMapsService and darkskyService to handle external API calls.  forecastRouter has been refactored to use both services.

### CODE REVIEW

File: services/darksky_service.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+var fetch = require('node-fetch');
```
Comment (alexander-mathieu) [inline services/darksky_service.js]: @bplantico What do you think about the implementation here?  Not as pretty as Rails, but it's doing the job.

File: services/google_maps_service.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+var fetch = require('node-fetch');
+
+class googleMapsService {
```
Comment (alexander-mathieu) [inline services/google_maps_service.js]: @bplantico

File: app.js [hunk 3]
```diff
@@ -1,3 +1,5 @@
+require('dotenv').config();
```
Comment (bplantico) [inline app.js]: You sure about this?

File: routes/api/v1/forecast.js [hunk 4]
```diff
@@ -16,24 +17,17 @@ router.get('/', (request, response) => {
...
-      fetch(`https://maps.googleapis.com/maps/api/geocode/json?key=${process.env.GOOGLE_MAPS_API_KEY}&address=${request.query.location}`)
-      .then(response => {
-        return response.json();
-      })
-      .then(data => {
-        let latLong = data.results[0].geometry.location;
-        let formattedLatLong = _formatLatLong(latLong);
-        return formattedLatLong
-      })
+      googleMapsService = new googleMapsService(request.query.location);
```
Comment (bplantico) [inline routes/api/v1/forecast.js]: Whoa! So clean.

File: services/darksky_service.js (hunk 1 above)
Comment (bplantico) [reply services/darksky_service.js]: This looks great actually.
Fetch may actually be ✨cleaner ✨ than Faraday, IMO....

File: services/google_maps_service.js (hunk 2 above)
Comment (bplantico) [reply services/google_maps_service.js]: If it looks like a Google Geocoding API service, and it talks like a Google Geocoding API service...

File: app.js (hunk 3 above)
Comment (alexander-mathieu) [reply app.js]: Oh, I'm sure.

**Pull request merged**